### PR TITLE
MAUI-3661-Added_API_Links_For_Migration_and_Getting_started_UG

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
## **Description**

- Added the API Links to the migration and Getting started UG

### CODE REVIEW

File: MAUI/Calendar/migration.md [hunk 1]
```diff
@@ -96,97 +96,97 @@ this.Content = calendar;
 <th>.NET MAUI SfCalendar</th>
 <th>Description</th></tr>
 <tr>
-<td>SelectionRange</td>
-<td>CalendarDateRange</td>
+<td>{{'[SelectionRange](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SelectionRange.html)'| markdownify }}</td>
```
Comment (Sasikumar3595) [inline MAUI/Calendar/migration.md:99]: Check the syntax is correct?

File: MAUI/Calendar/date-restrictions.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calend
 {% endhighlight %}
 {% highlight c# tabtitle="MainPage.xaml.cs" %}
 
-  this.Calendar.MinimumDateTime = DateTime.Now.AddDays(-5);
+  this.Calendar.MinimumDate = DateTime.Now.AddDays(-5);
```
Comment (Sasikumar3595) [inline MAUI/Calendar/date-restrictions.md:25]: Check the property is MinimumDate or MinimumDateTime

Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-restrictions.md:25]: Checked the property, it is MinimumDate

**Review by Sasikumar3595 — APPROVED** [review]
This is fine.

**Review by Nijamudeen-Mohamed — APPROVED** [review]
fine

**Pull request merged**